spring-2013/hw1/src/CreditCardValidator.hs

```diff
@@ -0,0 +1,15 @@
+module CreditCardValidator (toDigits, toDigitsRev, doubleEveryOther) where
+
+toDigits :: Integer -> [Integer]
+toDigits xs
```

Comment:
What about renaming `xs` to `n`, since it's a number, not a list?

And:

```haskell
toDigits n
  | n <= 0 = []
  | otherwise = map readChar $ show xs
```